pkg/network/protocols/kafka/api_version_counter.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	minSupportedAPIVersion = 1
-	maxSupportedAPIVersion = 11
+	maxSupportedAPIVersion = 12
```

Comment:
The PR does not prevent the next time it happens
I do recommend to populate the maximum supported versions from the kernel (via `types.go` file) and select the max

Reply:
Done